real_estate_module/models/estate_property.py

```diff
@@ -0,0 +1,40 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Real Estate Property'
+
+    name = fields.Char(required=True, string="Name")              
+    # create_uid = fields.Char(required=True)
+    # create_date = fields.Char(required=True)
+    postcode = fields.Char(string="Postcode")
+    description = fields.Text(string="Description")                    
```

Comment:
```suggestion
    description = fields.Text(string="Description")
```
Please make sure to apply it everywhere and in the future too there shouldn't be extra trailing space